plugins/woocommerce/tests/README.md

```diff
@@ -5,19 +5,24 @@ This document discusses unit tests. See [the e2e README](https://github.com/wooc
 ## Table of contents
 
 - [WooCommerce Tests](#woocommerce-tests)
-  - [Table of contents](#table-of-contents)
-  - [Initial Setup](#initial-setup)
-    - [MySQL database](#mysql-database)
-    - [Setup instructions](#setup-instructions)
-  - [Running Tests](#running-tests)
-    - [Troubleshooting](#troubleshooting)
-    - [Running tests in PHP 8](#running-tests-in-php-8)
-  - [Writing Tests](#writing-tests)
-  - [Automated Tests](#automated-tests)
-  - [Code Coverage](#code-coverage)
+    - [Table of contents](#table-of-contents)
+    - [Recommended Environment](#recommended-setup)
+    - [Manual Environment](#manual-setup)
+        - [MySQL database](#mysql-database)
+        - [Setup instructions](#setup-instructions)
+    - [Running Tests](#running-tests)
+        - [Troubleshooting](#troubleshooting)
+    - [Writing Tests](#writing-tests)
+    - [Automated Tests](#automated-tests)
+    - [Code Coverage](#code-coverage)
 
+## Recommended Setup
 
-## Initial Setup
+We provide support for running unit tests within a `wp-env` container. All you have to do is complete the [Getting Started Guide](https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce#getting-started) and have a functional `wp-env` environment.
```

Comment:
>All you have to do is

Is this a bit reductive? 

>and have a functional `wp-env` environment

I don't thing there are steps for this, like `npm -g i @wordpress/env`, so maybe reword a bit.

```suggestion
Complete the [Getting Started Guide](https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce#getting-started) and you will have a functional `wp-env` environment to run unit tests.
```